pizzeria-backend/wwwroot/index.html

```diff
@@ -2,19 +2,19 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="./favicon.ico" />
+    <link rel="icon" href="/assets/favicon-HwDZcgfE.ico" />
```

Comment:
All the files generated at buildtime should be in gitignore and not in the repo

Reply:
Done